data/json/npcs/robofac/robofac_intercom/robofac_intercom_trades.json

```diff
@@ -232,5 +232,263 @@
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_TRIAKIS",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_GOLDEN_MONOLITH",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_DELTA",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_QUADRAPHRACT_LEG",
+      "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_QUADRAPHRACT",
+      "TALK_ROBOFAC_INTERCOM_SELL_BOT_YRAX_TRIFACET",
+      "TALK_ROBOFAC_INTERCOM_SELL_BOT_YRAX_TRIAKIS",
+      "TALK_ROBOFAC_INTERCOM_SELL_BOT_GOLDEN_MONOLITH",
+      "TALK_ROBOFAC_INTERCOM_SELL_BOT_YRAX_DELTA"
+    ],
+    "type": "talk_topic",
+    "dynamic_line": "Beautiful; please put them into loading area.  We'll issue a reward depending on what you managed to scavenge.",
+    "responses": [
+      {
+        "text": "[Put in broken trifacet.]",
+        "condition": { "u_has_items": { "item": "broken_yrax_trifacet", "count": 1 } },
+        "effect": [
+          { "u_consume_item": "broken_yrax_trifacet", "count": 1, "popup": true },
+          { "math": [ "u_broken_yrax_trifacet_sold", "++" ] },
+          { "u_spawn_item": "RobofacCoin", "count": 1 }
+        ],
+        "topic": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_TRIFACET"
+      },
+      {
+        "text": "[Deliver a broken triakis drone.]",
+        "condition": {
+          "and": [
+            { "u_has_items": { "item": "broken_yrax_triakis", "count": 1 } },
+            { "math": [ "hub01_yrax_triakis_researched", "==", "1" ] }
+          ]
+        },
+        "effect": [
+          { "u_consume_item": "broken_yrax_triakis", "count": 1, "popup": true },
+          { "math": [ "u_broken_yrax_triakis_sold", "++" ] },
+          { "u_spend_cash": -1000 }
+        ],
+        "topic": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_YRAX_TRIAKIS"
+      },
+      {
+        "text": "[Deliver a broken monolith.]",
+        "condition": {
+          "and": [
+            { "u_has_items": { "item": "broken_golden_monolith", "count": 1 } },
+            { "math": [ "hub01_yrax_triakis_researched", "==", "1" ] }
+          ]
+        },
+        "effect": [
+          { "u_consume_item": "broken_golden_monolith", "count": 1, "popup": true },
+          { "math": [ "u_broken_golden_monolith_sold", "++" ] },
+          { "u_spawn_item": "RobofacCoin", "count": 7 }
+        ],
+        "topic": "TALK_ROBOFAC_INTERCOM_SELL_BROKEN_GOLDEN_MONOLITH"
+      },
+      {
+        "text": "[Deliver a broken delta drone.]",
+        "condition": {
+          "and": [
+            { "u_has_items": { "item": "broken_yrax_delta", "count": 1 } },
+            { "math": [ "hub01_yrax_delta_researched", "==", "1" ] }
+          ]
+        },
+        "effect": [
+          { "u_consume_item": "broken_yrax_delta", "count": 1, "popup": true },
+          { "math": [ "u_broken_yrax_delta_sold", "++" ] },
+          { "u_spend_cash": -200 }
+        ],
+        "topic": "TALK_ROBOFAC_INTERCOM_BROKEN_YRAX_DELTA"
+      },
+      {
+        "text": "[Put in broken triakias.]",
```

Comment:
This appears to be a repeat of the option above.

Reply:
It isn't top is for selling, this one is for research